abuild/Dockerfile

```diff
@@ -1,12 +1,15 @@
 # syntax=docker/dockerfile:experimental
-FROM alpine:3.13.7
+FROM alpine:3.14
+
 
 RUN --mount=type=cache,target=/var/cache/apk \ 
     --mount=type=cache,target=/etc/cache/apk \
     ln -s /var/cache/apk /etc/apk/cache && \
     apk add --update \
         alpine-sdk \
         coreutils \
+        sudo \
```

Comment:
I'm slightly curious as to why sudo is added?   When we log in we are root, so I'm wondering what value this adds or issue this fixes?

Reply:
@bseeger The ImageMagick build was failing before I added this. I'll remove it and run the build again to see the exact failure.

Reply:
Here's one of the test runs for the upgrade without `sudo`: https://github.com/jhu-idc/idc-isle-buildkit/runs/4412701426?check_suite_focus=true

Reply:
Weird, apparently Alpine doesn't generally ship with `sudo`, so I wonder how things worked before.

Reply:
Thanks for looking at that.